### ISSUE TITLE
Question about issue #71

### ISSUE THREAD

**Issue by Queist** [swsnu/swppfall2021#74]
issue #71 

Then, even article title is modified(?) but string is just same as before, (like type a letter 't' and delete the 't') back button should pop up confirm window?

I'm asking because I'm not sure I understand the answer correctly.

**Comment by kdh0102** [comment]
No, if the title is the same (even when you delete and write), we consider the title has not been modified.

**Comment by Queist** [comment]
Thanks for answering!
